src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -66,7 +66,8 @@ var HTMLDOMPropertyConfig = {
     // See http://schema.org/docs/gs.html
     itemScope: HAS_BOOLEAN_VALUE,
     // Facebook internal attribute. This is an object attribute that
-    // impliments a custom toString() method
+    // impliments a custom toString() method. Objects are not allowed as
```

Comment:
Man I need a git commit hook for impliments. Every. Single. Time. Fixed in aaafa7f